src/SOS/Strike/util.h

```diff
@@ -38,8 +33,11 @@ inline void RestoreSOToleranceState() {}
 #include "cordebug.h"
 #include "static_assert.h"
 #include <string>
+#include <releaseholder.h>
```

Comment:
Can we use the quote pattern for files we own please?

Reply:
Unless I'm mistaken, quotes means local files in the same directory as the source and <h> means to use the search directory paths.  The meaning has probably change over the years but I have always thought of it as that.

Reply:
Done